### DISCUSSION TITLE
Why does AGH show higher DNS server response times than other tests?

### DISCUSSION THREAD

**Discussion by beneix** [AdguardTeam/AdGuardHome#7344] [Q&A] [answered]
On the Dashboard, I see the average response times for upstream DNS servers for the last 24 hours. Lately, these have increased significantly (for the same upstreams) even though testing using drill (OPNSense's replacement for dig) against the same servers shows no change. Examples:
45.90.30.91 AGH 189 ms, drill 19ms
45.90.28.91 AGH 173 ms, drill 22ms
89.2.0.1 AGH 150 ms, drill 13ms
212.94.162.1 AGH 66 ms, drill 26ms

I have re-run the drill tests regularly over the last 10 hours and the results only vary by less than 5ms so even if you take into account that the AGH numbers are 24h averages, there seems to be something strange going on here.

This behaviour started in the last week, I have used AGH for years and not had this issue. Any ideas as to what could explain the difference?

**Comment by schzhn** [comment, marked as answer]
You could try the [edge release](https://github.com/AdguardTeam/AdGuardHome/blob/master/README.md#test-unstable-versions).
Related #6818

**Reply by beneix** [reply]
Thanks - I was going to wait for the stable release but since it is taking some time I was wondering, would you know how to install the edge release on OPNSense? I use the OPNSense AdGuardHome plugin, which normally handles updates through the web interface; it also has the option of updating through CLI but then it only looks for a new stable release. Is there a way of updating to the edge release?

**Reply by schzhn** [reply]
The stable release is near, as soon as issue #7314 is fixed.
